### PR TITLE
chore: improve onboarding

### PR DESCRIPTION
**Pull request by yordis**
Making the `CONTRIBUTING.md` to be a bit more replicable

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ docker compose up
 Initializing the db can be done with:
 
 ```sh
-mix ecto.setup
+make core-setup
```
Comment (yordis) [inline CONTRIBUTING.md:52]: The `cwd` is not `apps/core` and `ecto.setup` is not registered as an umbrella command.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ These apps will all depend on core, where most Plural business logic should live
 We use elixir's bamboo framework for templating and delivering emails, one benefit of which is it creates a local server to view in-progress emails.  You can get this set up by doing:
 
 ```bash
-mix ecto.create && mix ecto.migrate # ensure your dev db is set up
+make core-migrate # ensure your dev db is set up
```
Comment (yordis) [inline CONTRIBUTING.md:94]: The `cwd` is not `apps/core` and `ecto.create` and `ecto.migrate` is not registered as umbrella command.

File: apps/core/mix.exs [hunk 3]
```diff
@@ -131,7 +131,7 @@ defmodule Core.MixProject do
 
   defp aliases do
     [
-      "ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
+      "ecto.setup": ["ecto.create", "ecto.migrate"],
```
Comment (yordis) [inline apps/core/mix.exs:134]: the file "priv/repo/seeds.exs" does not exist, I am not sure if you meant for people to have their own `seeds.exs`

but it doesn't work as is

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ Plural's server side is written in elixir, and exposes a graphql api. The fronte
 To begin developing the web app, install [node](https://nodejs.org/en/download/) & [yarn](https://classic.yarnpkg.com/en/docs/getting-started/), then run:
 
 ```sh
-cd www && yarn install && cd ..
+make web-install
```
Comment (yordis) [inline CONTRIBUTING.md:14]: I dont see why not, less hassle with changing directories

**Review by michaeljguarino — APPROVED** [review]
thanks for the contribution!

**Pull request merged**